### DISCUSSION TITLE
How  to export pcap from Master node to out Security Onion

### DISCUSSION THREAD

**Discussion by admarnelson** [Security-Onion-Solutions/securityonion#6771] [Unsupported Versions] [answered]
Help how to do:

I would like to analysis specific pcap from and I would like use wireshark. the pcap are inside of Security Onion Master. How to export them?

**Comment by dougburks** [comment, marked as answer]
You can download pcap as shown at the bottom of this page:
https://docs.securityonion.net/en/2.3/pcap.html

**Reply by admarnelson** [reply]
Thanks Dougburks. so I tried to find some pcap in specific time to analysis them on wireshark but i couln't get it that link you showed. is there steo by step guide or video steps??
I also tried to get PCAPs from Master node (stenograph folder )with SCP tool but no sucessful even with full permition? Any help?


to be more specific I supose some workstation downloaded a malware that compromised a Network on 19th December now I want found this pcap

**Reply by dougburks** [reply]
> is there steo by step guide or video steps??

Pivoting to PCAP is a fairly common workflow and can be seen in many of our videos.

Take a look at our free Security Onion Essentials training:
https://securityonionsolutions.com/training/

and the other videos on our Youtube channel:
https://securityonion.net/youtube
